### PR TITLE
labels for authentication

### PR DESCRIPTION
**Pull request by rssag**

### CODE REVIEW

File: content/authentication/basic-settings.md [hunk 1]
```diff
@@ -85,15 +85,15 @@ If the user agent is not found in the list of trusted or forbidden user agents t
 {{< /c8y-admon-info >}}
 
 
-### OAI-Secure session configuration
+### OAI-Secure session configuration {#oaisecure-session-configuration}
```
Comment (BeateRixen) [inline content/authentication/basic-settings.md:88]: oaisecure > oai-secure

File: content/authentication/basic-settings.md [hunk 2]
```diff
@@ -85,15 +85,15 @@ If the user agent is not found in the list of trusted or forbidden user agents t
...
 
 OAI-Secure is a more secure alternative to the Basic Auth mode that also supports username and password login. In OAI-Secure mode the credentials in the initial request are exchanged for a JWT token that is set as a cookie in the web browser or returned in the response body. Based on the configuration OAI-Secure can support full session management or work as a standard JWT authentication where the user session lifetime is limited by the token expiration time.
 
-#### OAI-Secure without the configuration related to the session management (session configuration turned off)
+#### OAI-Secure without the configuration related to the session management (session configuration turned off) {#oaisecure-without-the-configuration-related-to-the-session-management-session-configuration-turned-off}
```
Comment (BeateRixen) [inline content/authentication/basic-settings.md:92]: oaisecure > oai-secure

File: content/authentication/basic-settings.md [hunk 3]
```diff
@@ -85,15 +85,15 @@ If the user agent is not found in the list of trusted or forbidden user agents t
...
 
 When there is no configuration related to the session, OAI-Secure issues a JWT token with a certain lifetime. If the token expires then the user is forced to re-login because token refresh is not supported. This behavior is very inconvenient for the user if the token lifetime is short because the user is forced to re-login frequently.  
 
-#### OAI-Secure with the configuration of the session management (session configuration turned on)
+#### OAI-Secure with the configuration of the session management (session configuration turned on) {#oaisecure-with-the-configuration-of-the-session-management-session-configuration-turned-on}
```
Comment (BeateRixen) [inline content/authentication/basic-settings.md:96]: oaisecure > oai-secure

File: content/authentication/basic-settings.md (hunk 1 above)
Comment (rssag) [inline content/authentication/basic-settings.md:88]:
```suggestion
### OAI-Secure session configuration {#oai-secure-session-configuration}
```

File: content/authentication/basic-settings.md (hunk 2 above)
Comment (rssag) [inline content/authentication/basic-settings.md:92]:
```suggestion
#### OAI-Secure without the configuration related to the session management (session configuration turned off) {#oai-secure-without-the-configuration-related-to-the-session-management-session-configuration-turned-off}
```

File: content/authentication/basic-settings.md (hunk 3 above)
Comment (rssag) [inline content/authentication/basic-settings.md:96]:
```suggestion
#### OAI-Secure with the configuration of the session management (session configuration turned on) {#oai-secure-with-the-configuration-of-the-session-management-session-configuration-turned-on}
```

File: content/authentication/basic-settings.md [hunk 4]
```diff
@@ -158,12 +158,12 @@ In such configurations, the idle timeout is in the range of 45 to 90 minutes, de
 During the session token renewal the previous token is revoked and a new one is provided. The parameter `renewal token delay` defines the delay used to make this process smooth and not disturbing for the user. The old token is still valid for this period (1 minute by default). This way both tokens, old and new, are accepted by {{< product-c8y-iot >}}. This parameter is only configurable on platform level and cannot be modified by the tenant administrator.
 
 
-### Token generation with OAI-Secure
+### Token generation with OAI-Secure {#token-generation-with-oaisecure}
```
Comment (rssag) [inline content/authentication/basic-settings.md:161]:
```suggestion
### Token generation with OAI-Secure {#token-generation-with-oai-secure}
```

File: content/authentication/basic-settings.md (hunk 1 above)
Comment (rssag) [reply content/authentication/basic-settings.md:88]: added

File: content/authentication/basic-settings.md (hunk 3 above)
Comment (rssag) [reply content/authentication/basic-settings.md:96]: added

File: content/authentication/basic-settings.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (rssag) [reply content/authentication/basic-settings.md:161]: added

File: content/authentication/basic-settings.md (hunk 2 above)
Comment (rssag) [reply content/authentication/basic-settings.md:92]: added

**Pull request merged**